appservice/src/intTest/groovy/nu/yona/server/BasicBuddyTest.groovy

```diff
@@ -412,7 +429,7 @@ class BasicBuddyTest extends AbstractAppServiceIntegrationTest
 		def dayActivityDetailBob = appService.getResourceWithPassword(dayActivityDetailUrlBob, bob.password)
 		assertResponseStatusOk(dayActivityDetailBob)
 		dayActivityDetailBob.responseData.date == YonaServer.toIsoDateString(goalConflictTime)
-		dayActivityDetailBob.responseData._links."yona:goal".href == goalBob.url
+		dayActivityDetailBob.responseData._links."yona:goal".href.startsWith(goalBob.url)
```

Comment:
Shouldn't we assert that the left hand side with stripped query string is equal to `goalBob.url`?

Reply:
Here it can be a simple comparison. Updated it.